<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>602</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Erfassung Kundengewichte und BMI</string>
  </property>
  <property name="windowIcon">
   <iconset resource="bmi.qrc">
    <normaloff>:/Icons/img/personenwaage.png</normaloff>:/Icons/img/personenwaage.png</iconset>
  </property>
  <property name="documentMode">
   <bool>true</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Werte erfassen</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" colspan="5">
          <widget class="QLineEdit" name="lineEdit_Name">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier den Namen des Kunden eintragen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Musterfrau, Frieda Annakatrin</string>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <spacer name="horizontalSpacer_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="9">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Erfassungsdatum:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Körpergröße:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QSpinBox" name="spinBox_Size">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier die Körpergröße des Kunden eintragen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <number>250</number>
           </property>
           <property name="value">
            <number>193</number>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>cm</string>
           </property>
          </widget>
         </item>
         <item row="1" column="4">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="5">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Körpergewicht:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="6">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_Weight">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>0</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier das aktuelle Gewicht des Kunden eintragen&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <double>260.990000000000009</double>
           </property>
           <property name="value">
            <double>103.200000000000003</double>
           </property>
          </widget>
         </item>
         <item row="1" column="7">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Kg</string>
           </property>
          </widget>
         </item>
         <item row="1" column="8">
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="9" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Gewicht der Steh-/Sitzhilfe:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="11">
          <widget class="QDoubleSpinBox" name="doubleSpinBox_RolliWeight">
           <property name="minimumSize">
            <size>
             <width>61</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Hier das aktuelle Gewicht der Steh- oder Sitzhilfe eintragen&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Hinweis:&lt;/span&gt; Therapiestuhl immer mit Fußstützen wiegen!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="maximum">
            <double>199.990000000000009</double>
           </property>
          </widget>
         </item>
         <item row="1" column="12">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Kg</string>
           </property>
          </widget>
         </item>
         <item row="0" column="10" colspan="2">
          <widget class="QDateTimeEdit" name="dateEdit">
           <property name="readOnly">
            <bool>false</bool>
           </property>
           <property name="displayFormat">
            <string>dd.MM.yyyy</string>
           </property>
           <property name="calendarPopup">
            <bool>true</bool>
           </property>
           <property name="timeSpec">
            <enum>Qt::TimeZone</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>480</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="btn_Berechnen">
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Berechnung starten. Ein Kunde kann erst nach erfolgter Berechnung&lt;/p&gt;&lt;p&gt;über die Schaltfläche &amp;quot;Übernehmen...&amp;quot; zum Report hinzugefügt werden.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Berechnen</string>
         </property>
         <property name="icon">
          <iconset resource="bmi.qrc">
           <normaloff>:/Icons/img/kcalc.png</normaloff>:/Icons/img/kcalc.png</iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>BMI:</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLCDNumber" name="lcdNumber">
             <property name="minimumSize">
              <size>
               <width>51</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>51</width>
               <height>23</height>
              </size>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Anzeige des errechneten BMI&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: qconicalgradient(cx:0, cy:0, angle:135, stop:0 rgba(255, 255, 0, 69), stop:0.375 rgba(255, 255, 0, 69), stop:0.423533 rgba(251, 255, 0, 145), stop:0.45 rgba(247, 255, 0, 208), stop:0.477581 rgba(255, 244, 71, 130), stop:0.518717 rgba(255, 218, 71, 130), stop:0.55 rgba(255, 255, 0, 255), stop:0.57754 rgba(255, 203, 0, 130), stop:0.625 rgba(255, 255, 0, 69), stop:1 rgba(255, 255, 0, 69));
color: rgb(0, 0, 0);</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_warnung">
           <property name="palette">
            <palette>
             <active>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </active>
             <inactive>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>255</red>
                 <green>0</green>
                 <blue>0</blue>
                </color>
               </brush>
              </colorrole>
             </inactive>
             <disabled>
              <colorrole role="Text">
               <brush brushstyle="SolidPattern">
                <color alpha="255">
                 <red>120</red>
                 <green>120</green>
                 <blue>120</blue>
                </color>
               </brush>
              </colorrole>
             </disabled>
            </palette>
           </property>
           <property name="text">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#ff0000;&quot;&gt;Warnung:&lt;/span&gt;&lt;span style=&quot; color:#ff0000;&quot;&gt; Mindest-BMI unterschritten!&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <spacer name="horizontalSpacer_6">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Expanding</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>10</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="3">
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QPushButton" name="btn_Neu">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Aktuelle Berechnung in den Report übernehmen und neue Eingabe vorbereiten&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Übernehmen und neue Erfassung</string>
             </property>
             <property name="icon">
              <iconset resource="bmi.qrc">
               <normaloff>:/Icons/img/user-group-new.png</normaloff>:/Icons/img/user-group-new.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Monatsreport - Druckausgabe</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0" colspan="3">
        <widget class="QTextEdit" name="textEdit_Report">
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOn</enum>
         </property>
         <property name="documentTitle">
          <string>Report</string>
         </property>
         <property name="lineWrapMode">
          <enum>QTextEdit::NoWrap</enum>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;title&gt;Report&lt;/title&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btn_Report_verwerfen">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="toolTip">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Einen bereits erstellten Report verwerfen.&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;Hinweis:&lt;/span&gt; Alle nicht gespeicherten Daten gehen verloren!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
         <property name="text">
          <string>Report verwerfen und neu erfassen</string>
         </property>
         <property name="icon">
          <iconset resource="bmi.qrc">
           <normaloff>:/Icons/img/recycle.png</normaloff>:/Icons/img/recycle.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>397</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="2">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <widget class="QPushButton" name="btn_Save">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Report speichern&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Speichern</string>
           </property>
           <property name="icon">
            <iconset resource="bmi.qrc">
             <normaloff>:/Icons/img/Save.png</normaloff>:/Icons/img/Save.png</iconset>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QPushButton" name="btn_Print">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="toolTip">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Report ausdrucken&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="text">
            <string>Drucken</string>
           </property>
           <property name="icon">
            <iconset resource="bmi.qrc">
             <normaloff>:/Icons/img/print_printer.png</normaloff>:/Icons/img/print_printer.png</iconset>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>602</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuDatei">
    <property name="title">
     <string>Datei</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="actionBerechnen"/>
    <addaction name="actionReport_verwerfen_und_neu_erfassen"/>
    <addaction name="separator"/>
    <addaction name="actionDrucken"/>
    <addaction name="separator"/>
    <addaction name="actionReport_importieren"/>
    <addaction name="actionSpeichern"/>
    <addaction name="actionReport_speichern_unter"/>
    <addaction name="separator"/>
    <addaction name="actionEnde"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionProgramminformation"/>
    <addaction name="actionInfo_zu_Qt"/>
   </widget>
   <addaction name="menuDatei"/>
   <addaction name="menuHilfe"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBerechnen"/>
   <addaction name="actionNeu"/>
   <addaction name="actionReport_verwerfen_und_neu_erfassen"/>
   <addaction name="separator"/>
   <addaction name="actionSpeichern"/>
   <addaction name="actionReport_speichern_unter"/>
   <addaction name="actionReport_importieren"/>
   <addaction name="separator"/>
   <addaction name="actionDrucken"/>
   <addaction name="separator"/>
   <addaction name="actionProgramminformation"/>
   <addaction name="actionInfo_zu_Qt"/>
   <addaction name="separator"/>
   <addaction name="actionEnde"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNeu">
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/user-group-new.png</normaloff>:/Icons/img/user-group-new.png</iconset>
   </property>
   <property name="text">
    <string>Übernehmen und neue Erfassung</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionBerechnen">
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/kcalc.png</normaloff>:/Icons/img/kcalc.png</iconset>
   </property>
   <property name="text">
    <string>Berechnen</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionEnde">
   <property name="icon">
    <iconset theme="Waage.png" resource="bmi.qrc">
     <normaloff>:/Icons/img/exit (1).png</normaloff>:/Icons/img/exit (1).png</iconset>
   </property>
   <property name="text">
    <string>Ende</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionProgramminformation">
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/Info.png</normaloff>:/Icons/img/Info.png</iconset>
   </property>
   <property name="text">
    <string>Programminformation</string>
   </property>
  </action>
  <action name="actionInfo_zu_Qt">
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/ktip.png</normaloff>:/Icons/img/ktip.png</iconset>
   </property>
   <property name="text">
    <string>Info zu Qt</string>
   </property>
  </action>
  <action name="actionSpeichern">
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/Save.png</normaloff>:/Icons/img/Save.png</iconset>
   </property>
   <property name="text">
    <string>Report speichern</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionDrucken">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/print_printer.png</normaloff>:/Icons/img/print_printer.png</iconset>
   </property>
   <property name="text">
    <string>Report drucken...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionReport_speichern_unter">
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/SaveAs.png</normaloff>:/Icons/img/SaveAs.png</iconset>
   </property>
   <property name="text">
    <string>Report speichern unter...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionReport_importieren">
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/OpenFile.png</normaloff>:/Icons/img/OpenFile.png</iconset>
   </property>
   <property name="text">
    <string>Report  importieren</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionReport_verwerfen_und_neu_erfassen">
   <property name="icon">
    <iconset resource="bmi.qrc">
     <normaloff>:/Icons/img/recycle.png</normaloff>:/Icons/img/recycle.png</iconset>
   </property>
   <property name="text">
    <string>Report verwerfen und neu erfassen</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="bmi.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionEnde</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>300</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionBerechnen</sender>
   <signal>triggered()</signal>
   <receiver>btn_Berechnen</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>543</x>
     <y>152</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionNeu</sender>
   <signal>triggered()</signal>
   <receiver>btn_Neu</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>486</x>
     <y>193</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionReport_verwerfen_und_neu_erfassen</sender>
   <signal>triggered()</signal>
   <receiver>btn_Report_verwerfen</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btn_Save</sender>
   <signal>clicked()</signal>
   <receiver>actionSpeichern</receiver>
   <slot>trigger()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>463</x>
     <y>434</y>
    </hint>
    <hint type="destinationlabel">
     <x>463</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDrucken</sender>
   <signal>triggered()</signal>
   <receiver>btn_Print</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>544</x>
     <y>434</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
